utils/update_checkout/update-checkout-config.json

```
@@ -113,6 +113,9 @@
         },
         "zlib": {
           "remote": { "id": "madler/zlib" }
+        },
+        "mimalloc": {
+          "remote": { "id": "microsoft/mimalloc" }
```

Can you add
```suggestion
          "remote": { "id": "microsoft/mimalloc" },
          "platforms": [ "Windows" ]
```

This way we're only downloading it on Windows.

---

I think that is best done in a follow up if we want to remove some of these repositories from being checked out (e.g. playground-support).

---

I'm fine if there's a follow-up for the others, but this is new stuff.

---

Doing that as a single pass is better IMO.